playbook/lib/playbook/forms/builder/typeahead_field.rb

```diff
@@ -5,6 +5,7 @@ module Forms
     class Builder
       def typeahead(name, _options = {}, _html_options = {}, props: {})
         props[:name] = name
+        props[:label] = props[:label] == true ? name.to_s.underscore.humanize : props[:label]
```

Comment:
If we did something similar to this:

https://github.com/powerhome/playbook/blob/244697cbb9c2de700a3fc7422723415cec66f73c/playbook/lib/playbook/forms/builder/form_field_builder.rb#L9

Then we would get the benefit of Rails inflections and translations for the files name. That is a lot of features that can greatly improve the usability of this kit.

Reply:
I had tried to get this construction to work for a while but was not able to get label click-to-focus functionality to work with typeahead I think because of the nature of the embedded/under the kit input differing from how other kits have it. Can definitely give it another shot (may require a bit more kit refactoring).